### PR TITLE
EES-4059 update release page sidebar links

### PR DESCRIPTION
**Pull request by amyb-hiveit**
Updates the links in the release page sidebar in the public frontend and admin.

### CODE REVIEW

File: src/explore-education-statistics-admin/src/pages/release/content/components/RelatedPagesSection.tsx [hunk 1]
```diff
@@ -93,10 +93,7 @@ const RelatedPagesSection = ({ release }: Props) => {
     <>
       {(editingMode === 'edit' || links.length > 0) && (
         <>
-          <h3
-            className="govuk-heading-s govuk-!-margin-top-6"
-            id="related-pages"
-          >
+          <h3 className="govuk-heading-s " id="related-pages">
```
Comment (luke-h1) [inline src/explore-education-statistics-admin/src/pages/release/content/components/RelatedPagesSection.tsx:96]:
```suggestion
          <h3 className="govuk-heading-s" id="related-pages">
```

File: src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx [hunk 2]
```diff
@@ -224,11 +198,44 @@ const ReleaseContent = () => {
...
+                <li>
+                  <a href="#help-and-support">Help and support</a>
+                </li>
               </ul>
             </nav>
 
-            <h2 className="govuk-heading-m">Supporting information</h2>
-            <ul className="govuk-list govuk-list--spaced govuk-!-margin-bottom-0">
+            <h2 className="govuk-heading-s">Related information</h2>
+            <ul className="govuk-list ">
```
Comment (luke-h1) [inline src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx:219]:
```suggestion
            <ul className="govuk-list">
```

File: src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx [hunk 3]
```diff
@@ -256,16 +263,14 @@ const ReleaseContent = () => {
 
             {!!releaseCount && (
               <>
-                <h3
-                  className="govuk-heading-s govuk-!-margin-top-6"
-                  id="past-releases"
-                >
+                <h3 className="govuk-heading-s " id="past-releases">
```
Comment (luke-h1) [inline src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx:266]:
```suggestion
                <h3 className="govuk-heading-s" id="past-releases">
```

File: src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx [hunk 4]
```diff
@@ -297,12 +302,12 @@ const ReleaseContent = () => {
             {allMethodologies.length > 0 && (
               <>
                 <h3
-                  className="govuk-heading-s govuk-!-margin-top-6"
+                  className="govuk-heading-sgovuk-!-padding-top-0"
```
Comment (luke-h1) [inline src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx:305]: Is this classname correct? (govuk-heading-sgovuk-!-padding-top-0). Just wondering if it should be `govuk-heading-s govuk-!-padding-top-0` instead?

Comment (luke-h1) [inline src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx:305]: Is this classname correct? (govuk-heading-sgovuk-!-padding-top-0). Just wondering if it should be `govuk-heading-s govuk-!-padding-top-0` instead?

File: src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx [hunk 5]
```diff
@@ -297,12 +302,12 @@ const ReleaseContent = () => {
...
                   id="methodologies"
                 >
                   Methodologies
                 </h3>
-                <ul className="govuk-list govuk-list--spaced govuk-!-margin-bottom-0">
+                <ul className="govuk-list ">
```
Comment (luke-h1) [inline src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx:310]:
```suggestion
                <ul className="govuk-list">
```

File: src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx [hunk 6]
```diff
@@ -228,66 +228,96 @@ const PublicationReleasePage: NextPage<Props> = ({ release }) => {
...
+                    Help and support
+                  </a>
+                </li>
               </ul>
             </nav>
-            <h2 className="govuk-heading-m">Supporting information</h2>
-            <ul className="govuk-list govuk-list--spaced govuk-!-margin-bottom-0">
+
+            <h2 className="govuk-heading-s">Related information</h2>
+            <ul className="govuk-list ">
```
Comment (luke-h1) [inline src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx:308]:
```suggestion
            <ul className="govuk-list">
```

File: src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx (hunk 4 above)
Comment (amyb-hiveit) [reply src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx]: Nope! It should have a space between them as you suggest.

**Review by luke-h1 — APPROVED** [review]

**Pull request merged**